src/Microphysics1M.jl

```diff
@@ -46,6 +46,35 @@ export accretion_snow_rain
 export evaporation_sublimation
 export snow_melt
 
+"""
+    logistic_function_integral(x, x_0, k)
+ 
```

Comment:
Lets put that function into common (https://github.com/CliMA/CloudMicrophysics.jl/blob/main/src/Common.jl) This way we won't have to define it twice.

Reply:
logistic_function and logistic_function_integral are two different function, but I agree that putting both of them in Common is cleaner